software/programming_instructions.md

```diff
@@ -28,13 +28,16 @@ The quickest way to get your EuroPi flashed with the latest firmware is to head
 
 ## Setting Up
```

Comment:
[Suggestion]
Potentially change "Setting Up" to "Pico Set Up" to match the "Pico 2 Set Up" title lower down. This way it's clearer that the two are different versions of the same step in the overall module setup

Reply:
Done.